.github/workflows/hello-charm-tests.yaml

```diff
@@ -22,6 +24,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: ${{ matrix.charm-repo }}
+          ref: ${{ matrix.revision }}
```

Comment:
This is "tag" some places and "revision" others. But we may sometimes set "tag" to a revision. I think we should just use "ref" everywhere.

Reply:
I've changed the pins back to commits